roles/acme_certificate/tasks/dns-inwx-create.yml

```
@@ -0,0 +1,27 @@
+---
+# Create DNS challenges for DNS provider INWX
+
+- name: Creating challenge DNS entries for {{ ', '.join(acme_certificate_domains + acme_certificate_ips) }} via INWX
+  inwx.collection.dns:
+    state: present
+    domain: "{{ item.0.key | community.dns.get_registrable_domain }}"
+    type: TXT
+    record: "{{ item.0.key | community.dns.remove_registrable_domain }}"
+    value: "{{ item.1 }}"
+    solo: "{{ 'true' if ansible_loop.first else 'false' }}"
```

This can be simplified:
```suggestion
    solo: "{{ ansible_loop.first }}"
```